pandas/tests/test_multilevel.py

```diff
@@ -519,7 +519,8 @@ def test_xs_level_multiple(self):
 a   q   20      4     0.4473  1.4152  0.2834  1.00661  0.1744
 x   q   30      3    -0.6662 -0.5243 -0.3580  0.89145  2.5838"""
 
-        df = read_table(StringIO(text), sep=r'\s+', engine='python')
+        with tm.assert_produces_warning(FutureWarning):
```

Comment:
anywhere non-essentiall, just change to use read_csv

Reply:
Did my best to find non-essential places and do this.